staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -196,16 +198,20 @@ type watchCache struct {
 
 	// For testing cache interval invalidation.
 	indexValidator indexValidator
+
+	requestBookmark BookmarkRequester
 }
 
+type BookmarkRequester func(ctx context.Context) error
+
 func newWatchCache(
 	keyFunc func(runtime.Object) (string, error),
 	eventHandler func(*watchCacheEvent),
 	getAttrsFunc func(runtime.Object) (labels.Set, fields.Set, error),
 	versioner storage.Versioner,
 	indexers *cache.Indexers,
 	clock clock.Clock,
-	groupResource schema.GroupResource) *watchCache {
+	groupResource schema.GroupResource, requestBookmark BookmarkRequester) *watchCache {
```

Comment:
nit: split the line (for consistency with everything else - one param per line)

Reply:
Done